commands/run.go

```diff
@@ -0,0 +1,207 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"os/exec"
+
+	"github.com/openfaas/faas-cli/stack"
+	"github.com/spf13/cobra"
+)
+
+const localSecretsDir = ".secrets"
+
+func init() {
+	faasCmd.AddCommand(newRunCmd())
+}
+
+type runOptions struct {
+	print    bool
+	port     int
+	network  string
+	extraEnv map[string]string
+	output   io.Writer
+	err      io.Writer
+}
+
+func newRunCmd() *cobra.Command {
+	opts := runOptions{}
+
+	cmd := &cobra.Command{
+		Use:   `run NAME -f YAML_FILE`,
```

Comment:
For first-time users of OpenFaaS `faas-cli run` sounds like it runs the function, and not that it's for local testing only.

I think `local-run` (despite being longer) with an alias or `lr` perhaps? Would be more intuitive